homeassistant/components/recorder/purge.py

```diff
@@ -15,20 +13,15 @@
 from .const import MAX_ROWS_TO_PURGE
 from .models import Events, RecorderRuns, States
 from .repack import repack_database
-from .util import session_scope
+from .util import retriable_database_job, session_scope
```

Comment:
should this be retryable ?

Reply:
Yes, we want to retry the purge if it fails due to a temporary issue? @bdraco can you comment?

Reply:
We don't want to throw when there is a transient network issue, the MySQL server is restarted, a connection is dropped, the server side timeout reached and the connection pool has a stale connection, etc.

The retries guard against this and its status quo every where we write to the database.

Reply:
Sorry, I meant the spelling 🤣

Reply:
https://english.stackexchange.com/questions/305273/retriable-or-retryable#:~:text=Retryable%20is%20the%20more%20correct,by%20a%20court%20of%20law